ogre_mesh_viewer.py

```diff
@@ -290,17 +292,27 @@ def preRenderTargetUpdate(self, evt):
 
         lod_count = mesh.getNumLodLevels()
         if lod_count > 1 and ImGui.CollapsingHeader("LOD levels"):
-            entity.setMeshLodBias(1)  # reset LOD override
+            entity.setMeshLodBias(1, self.lod_current_idx, self.lod_current_idx)  # reset LOD override
             strategy = mesh.getLodStrategy().getName()
             curr_idx = entity.getCurrentLodIndex()
             ImGui.Text("Strategy: {}".format(strategy))
+
+            lod_levels = []
             for i in range(lod_count):
-                txt = "Base Mesh" if i == 0 else "Level {}: {:.2f}".format(i, mesh.getLodLevel(i).userValue)
-                ImGui.Bullet()
-                ImGui.Selectable(txt, i == curr_idx)
```

Comment:
what this was doing before is that the respective LOD in the list became active as the camera moved away. This should still work.
I think we should use `lod_current_idx=-1` for "no LOD override" and reset it to -1 if the user clicks on Base Mesh.

Reply:
Oh, I didn't know about that.

Reply:
What about this?

If the user moves the camera, then whatever the user has selected gets reset to what should be by distance.

Reply:
on the other hand.. was not the whole point here that the user can move the camera to inspect the LOD? Like if he wants to know whether the LOD distance needs to be decreased?

Reply:
I was thinking more in terms of being able to freeze the selected LOD in order to see around it.

Not in terms of moving the camera in and out.

Anyway, I think that the latest changes solve the problem.